front/app/containers/Admin/messaging/CustomEmails/All/index.tsx

```diff
@@ -41,17 +43,21 @@ const NoCampaignsDescription = styled.p`
   max-width: 450px;
 `;
 
-interface DataProps extends GetCampaignsChildProps {}
+const CustomEmails = () => {
+  const location = useLocation();
+  const query = parse(location.search, { ignoreQueryPrefix: true });
+  const pageNumber = query.pageNumber;
+  const pageNumberInt = pageNumber ? parseInt(pageNumber as string, 10) : 1;
 
-interface Props extends DataProps {}
+  const { data } = useCampaigns({
```

Comment:
```suggestion
  const { data: campaigns } = useCampaigns({
```

This removes the need for 
`  const campaigns = data.data;
`on line 59.